docusaurus/docs/develop/path/path_config.md

```diff
@@ -282,9 +282,11 @@ In particular, allows specifying server parameters for how the gateway handles i
 
 The service ID must be provided here.
 
-| Field         | Type          | Required | Default | Description                                                                 |
-| ------------- | ------------- | -------- | ------- | --------------------------------------------------------------------------- |
-| `service_ids` | array[string] | No       | -       | List of service IDs for which the Quality of Service (QoS) logic will apply |
+| Field                        | Type          | Required | Default   | Description                                                                          |
```

Comment:
Can you think of a separate utility that ensures the .md file matches the configs?

I'm even thinking a go/python script, that's triggered via a makefile.

Maybe this will act as inspiration: https://github.com/pokt-network/poktroll/blob/bb631a2c34bf426f7ca6ee0182dfbe979e0e8c3e/tools/scripts/docusaurus/generate_docs_params.go

Reply:
Just ACKing that I've seen this. Looked into implementing this briefly; IMO the amount of work that this would take to make a reliable script for this vs the amount of time it takes to manually update the docs on the rare occasion we add a new config field makes this work is not on the "critical path" at this moment.